modules/javafx.controls/src/main/java/javafx/scene/control/TableUtil.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2012, 2016, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2012, 2016, 2022 Oracle and/or its affiliates. All rights reserved.
```

Comment:
Not quite. There should be at most two years, and there must be a comma after each one. So:

```
 * Copyright (c) 2012, 2022, Oracle and/or its affiliates. All rights reserved.
```

Reply:
@kevinrushforth Sorry that I am not aware of this standard. Is it that the two years is some thing like the first is the initial modification year and the second is the latest modification year?

Reply:
Yes, exactly.